### ISSUE TITLE
Line breaks are removed on string inject

### ISSUE THREAD

**Issue by debrice** [klei/gulp-inject#101]
Hi,

I'm having issue when I do a direct string injection

```coffee
gulp.src("#{COMPILED_EXAMPLES}/template.html")
  # inject the Jade example in <pre><code></code></pre>
  .pipe inject gulp.src("#{example_folder}/example/*.jade"),
    starttag: '<!-- inject:jade -->'
    transform: (filePath, file) ->
      return file.contents.toString('utf8')
```

All the line breaks are removed, making a multiline example a very long one liner. The changes are done by this line https://github.com/klei/gulp-inject/blob/master/src/inject/index.js#L195 . 

I'm not sure I understand why the content of the string is altered here by removing the line breaks and breaking the string in an array of lines.

**Comment by joakimbeng** [comment]
@debrice sorry for the late response...

> I'm not sure I understand why the content of the string is altered here...

Actually I don't remember why I made the plugin do so either. It looks like it could safely be removed without breaking anything.
I'll have to test it and get back to you.

**Comment by joakimbeng** [comment]
@debrice fixed this in `1.4.0`.

**Comment by debrice** [comment]
@joakimbeng thank you very much.
